packages/react/src/components/hooks/useLayouts.ts

```diff
@@ -0,0 +1,32 @@
+import { Video, VideoPositions } from "@signalwire/js";
+import { useEffect, useState } from "react";
+
+export default function useLayouts(roomSession: Video.RoomSession | null) {
```

Comment:
could we add a `/**` comment above each of the hooks? It could contain minimal information about what the hook does, so that it will show up in intellisense for people.